build/root/Makefile

```diff
@@ -46,12 +46,12 @@ GENERATED_FILE_PREFIX := zz_generated.
 # Metadata for driving the build lives here.
 META_DIR := .make
 
-# Our build flags.
-# TODO(thockin): it would be nice to just use the native flags.  Can we EOL
-#                these "wrapper" flags?
-KUBE_GOFLAGS := $(GOFLAGS)
-KUBE_GOLDFLAGS := $(GOLDFLAGS)
-KUBE_GOGCFLAGS = $(GOGCFLAGS)
+ifdef KUBE_GOFLAGS
+$(info KUBE_GOFLAGS is deprecated. Please use GOFLAGS instead)
+ifndef GOFLAGS
+GOFLAGS := $(KUBE_GOFLAGS)
```

Comment:
@thockin - i was thinking that if someone sets both GOFLAGS and KUBE_GOFLAGS to the same thing say `-v` then we will have `-v -v` causing failure. So trying to avoid that scenario